crypto/ube/ube_test.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include  <gtest/gtest.h>
```

Comment:
Looking at the [coverage report](https://app.codecov.io/gh/aws/aws-lc/pull/1801?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=aws) can you add tests for:
1. ube_failed and short circuit behavior
2. UBE and race condition with another thread already updating the value

Reply:
I plan way more extensive testing down the line, quite a bit more than we currently have.

The test you mention require more extensive facilities. I want to add those in separate PRs to avoid doing many things at the same time.